bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -353,14 +353,28 @@ mod tests {
         assert!(r.get_create_address().is_none());
     }
 
+    // Test-specific helper to dispose of execution results in unit tests
+    extern "C" fn test_result_dispose(result: *const ffi::evmc_result) {
+        unsafe {
+            if !result.is_null() {
+                let owned = *result;
+                let owned_retbuf = Vec::from_raw_parts(
```

Comment:
Cannot all this just work with `drop(Box::from_raw(result))` ?

Reply:
A box taking ownership of a pointer will only drop the pointer.
You can do a boxed slice but that is more or less the same thing as `Vec` with extra steps.